src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -26,20 +26,21 @@ public class Section {
     @JoinColumn(name = "down_station_id")
     private Station downStation;
 
-    private Integer distance;
+    @Embedded
+    private SectionDistance sectionDistance;
 
-    private Section(Line line, Station upStation, Station downStation, int distance) {
+    private Section(Line line, Station upStation, Station downStation, SectionDistance sectionDistance) {
         this.line = line;
         this.upStation = upStation;
         this.downStation = downStation;
-        this.distance = distance;
+        this.sectionDistance = sectionDistance;
     }
 
-    public static Section createSection(Line line, Station upStation, Station downStation, Integer distance) {
+    public static Section createSection(Line line, Station upStation, Station downStation, SectionDistance sectionDistance) {
         assert line != null;
         assert upStation != null;
         assert downStation != null;
 
-        return new Section(line, upStation, downStation, distance);
+        return new Section(line, upStation, downStation, sectionDistance);
```

Comment:
`Section.createSection()`을 호출할 때마다 매번 `SectionDistance` 인스턴스를 직접 생성해서 넘겨주는 대신에,
구간의 거리 값을 전달하면 내부에서 `SectionDistance` 인스턴스를 생성하도록 해보는 건 어떨까요?
```suggestion
        return new Section(line, upStation, downStation, new SectionDistance(distance));
```

Reply:
실제로 좀 고민을 했던 부분인데, 타입으로 해당 값이 어떤 용도로 사용되는 지를 알려야 한다는 생각에  `SectionDistance` 타입으로 변경을 했었습니다. 지금 와서 보면 굳이 내부에서 무엇을 어떻게 하여 거리 값을 사용하는지 굳이 알고 있을 필요는 없을 것 같네요.